osu.Game/Overlays/Mods/ModCustomisationPanel.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Input.Events;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Input.Bindings;
+using osu.Game.Rulesets.Mods;
+using osuTK;
+
+namespace osu.Game.Overlays.Mods
+{
+    public partial class ModCustomisationPanel : OverlayContainer, IKeyBindingHandler<GlobalAction>
+    {
+        private const float header_height = 42f;
+        private const float content_vertical_padding = 20f;
+
+        private Container content = null!;
+        private OsuScrollContainer scrollContainer = null!;
+        private FillFlowContainer sectionsFlow = null!;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; } = null!;
+
+        public readonly BindableBool Enabled = new BindableBool();
+
+        public readonly BindableBool Expanded = new BindableBool();
+
+        public Bindable<IReadOnlyList<Mod>> SelectedMods { get; } = new Bindable<IReadOnlyList<Mod>>(Array.Empty<Mod>());
+
+        public override bool HandlePositionalInput => Expanded.Value;
+
+        public override bool HandleNonPositionalInput => Expanded.Value;
+
+        protected override bool BlockPositionalInput => true;
```

Comment:
slightly bothered by this, why is this not `Expanded.Value` too?

Reply:
Hopefully https://github.com/ppy/osu/pull/28553/commits/86583898548c83e3211b6c10f653b88382ada954 should explain this. TL;DR this override is redundant and even then `HandlePositionalInput` controls whether the block will be effective or not. `BlockPositionalInput` is not supposed to return a variable anyway, it has to be constant.